master/buildbot/master.py

```diff
@@ -228,7 +238,7 @@ def startService(self):
                 # run the master.cfg in thread, so that it can use blocking code
                 self.config = yield threads.deferToThreadPool(
```

Comment:
That seems like a reasonable change, but larger than I want to do at this point. The current change makes it easy to test code without needing to generate python code and evaluate it, just to pass an existing dictionary around.

In particular, doing that requires a fair amount of refactoring tests to support deferreds.